src/contexts/W3iContext/hooks/pushHooks.ts

```diff
@@ -12,6 +13,8 @@ export const usePushState = (w3iProxy: Web3InboxProxy, proxyReady: boolean) => {
     PushClientTypes.PushSubscription[]
   >([])
 
+  const { pathname } = useLocation()
```

Comment:
Needed? Don't see this being used in the diffs but maybe I'm missing something

Reply:
it's used to allow in the dependency array in the below `useEffect`. This is to allow the modal to retrigger during navigation to `/widget/subscribe`

Reply:
My bad, didn't realise it was a dep array at first 🤦‍♂️